pkg/buildpack/buildpack_test.go

```diff
@@ -1002,3 +1037,11 @@ func assertWindowsBuildpacksToTar(t *testing.T, actual []buildpack.ModuleTar, ex
 		h.AssertTrue(t, found)
 	}
 }
+
+type blobDir struct {
+	path string
+}
+
+func (b *blobDir) Open() (io.ReadCloser, error) {
+	return archive.ReadDirAsTar(b.path, ".", 0, 0, -1, true, false, nil), nil
```

Comment:
I need a `Blob` which source is a folder, so basically I took the same method call from [here](https://github.com/buildpacks/pack/blob/main/pkg/blob/blob.go#L33)

Reply:
My fault, there was already a constructor!